service/src/main/java/uk/gov/hmcts/reform/fpl/events/LegalCounsellorAdded.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+import org.apache.commons.lang3.tuple.Pair;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.LegalCounsellor;
+
+@RequiredArgsConstructor
+@Value
+public class LegalCounsellorAdded {
+    CaseData caseData;
+    Pair<String, LegalCounsellor> legalCounsellor;
```

Comment:
vv minor any reason for storing as a pair and not as separate values?

Reply:
I just though it would be more obvious what that the id relates to this user.
But I can just as well see the benefit of having it separate.
What do you think?

Reply:
I am fine with either tbh

Reply:
Ok. Will keep it as it is, then.